### ISSUE TITLE
RuntimeError: Evaluation error: replacement has length zero

### ISSUE THREAD

**Issue by LouisAsh** [lorenzwalthert/KerasMisc#1]
Just installed your package, but while the toy example you describe with the Boston data works, when I try to use the callback with other models I get the error:

> Error in py_call_impl(callable, dots$args, dots$keywords) : 
>   RuntimeError: Evaluation error: replacement has length zero.
> 
> Detailed traceback:
>   File "/home/ash/.local/lib/python3.6/site-packages/tensorflow/python/keras/engine/training.py", line 1145, in fit
>     callbacks.on_epoch_end(epoch, epoch_logs)
>   File "/home/ash/.local/lib/python3.6/site-packages/tensorflow/python/keras/callbacks.py", line 432, in on_epoch_end
>     callback.on_epoch_end(epoch, numpy_logs)
>   File "/home/ash/R/x86_64-pc-linux-gnu-library/3.6/keras/python/kerastools/callback.py", line 5
> 6, in on_epoch_end
>     self.r_on_epoch_end(epoch, logs)
>   File "home/ash/R/x86_64-pc-linux-gnu-library/3.6/reticulate/python/rpytools/call.py", line 21
> , in python_function
>     raise RuntimeError(res[kErrorKey])
> In addition: Warning message:
> In rep(no, length.out = len) : 'x' is NULL so the result will be NULL

Would you happen to know a fix for that?

**Comment by lorenzwalthert** [comment]
Can you please create a [reprex](https://reprex.tidyverse.org/articles/articles/learn-reprex.html)? Otherwise, I can't debug this.

**Comment by LouisAsh** [comment]
I think I have identified the problem: your implementation seems to break when a model has more than 1 output.

Here I send attached minimally reproducible example that throws me the error.

[min-example-2.txt](https://github.com/lorenzwalthert/KerasMisc/files/6400200/min-example-2.txt)

**Comment by LouisAsh** [comment]
@lorenzwalthert Just so you know, I did try other examples with and without multiple outcomes. Indeed, all attempts with single output nets worked well and all attempts with multiple outputs nets gave the same error. So it does seem to be related to that.

Also, while I did not create a reprex, the file I sent has a fully reproducible example (with data generated within it). Do let me know if that suffices to help you spot the issue.

Many thanks again for your package.
